beacon-chain/core/validators/validator.go

```diff
@@ -634,3 +634,99 @@ func isActiveValidator(validator *pb.ValidatorRecord) bool {
 	return validator.Status == pb.ValidatorRecord_ACTIVE_PENDING_EXIT ||
 		validator.Status == pb.ValidatorRecord_ACTIVE
 }
+
+// activateValidator takes in validator index and activates
```

Comment:
Fyi I didn't add psesus code to the comments because these aren't exported. Lmk if you guys still want it